agent/hcp/telemetry/custom_metrics.go

```diff
@@ -0,0 +1,14 @@
+package telemetry
```

Comment:
Keys for custom metrics for the operations of the sink. These metrics do not need to be registered in consul as a whole (e.g. for prometheus) as they will only exist when the HCP sink is created, so only the OTELSink / Inmemsink will be receiving these metrics.

Is a separate file overkill?